### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,70 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None) -> None:
```
Comment (EdAlekseiev) [inline app/main.py:10]: You must specify the inner type of `list`. For example, if your list contains `str` elements, so type hint must be `list[str]`. Also, if your argument can be None, you must show it in type hint. `list[str] | None`

Comment (EdAlekseiev) [inline app/main.py:10]:
```suggestion
                 coords: list[int] | None = None) -> None:
```

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,70 @@ class Cargo:
...
+        self.name = name
+        self.weight = weight
+
+        if coords is None:
+            coords = [0, 0]
```
Comment (EdAlekseiev) [inline app/main.py:15]: Use ternary operator

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,70 @@ class Cargo:
...
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None) -> None:
```
Comment (EdAlekseiev) [inline app/main.py:39]: The same issue as above

File: app/main.py [hunk 4]
```diff
@@ -2,4 +2,70 @@ class Cargo:
...
+        super().__init__(name, weight, coords)
+
+        if coords is None:
+            coords = [0, 0, 0]
```
Comment (EdAlekseiev) [inline app/main.py:43]: Use ternary operator

File: app/main.py [hunk 5]
```diff
@@ -2,4 +2,70 @@ class Cargo:
...
+
+
+class DeliveryDrone(FlyingRobot):
+
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 max_load_weight: int,
+                 current_load: int,
+                 coords: list = None) -> None:
```
Comment (EdAlekseiev) [inline app/main.py:61]: The same issue as above

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**